### PR TITLE
docs: Fix non-working code example in breadcrumb browserHistory

### LINKED ISSUE #47181: Breadcrumb doc does not include working example
### Reproduction link

[https://ant.design/components/breadcrumb#use-with-browserhistory](https://ant.design/components/breadcrumb#use-with-browserhistory)

### Steps to reproduce

open doc page, scroll to section, see no working example

### What is expected?

a working example

### What is actually happening?

no working example

| Environment | Info |
| --- | --- |
| antd | 5.13.2 |
| React | react v 18 |
| System | Win 10 |
| Browser | Edge 120.0.2210.144 |

---
please create a working example

### PR DESCRIPTION
**Pull request by rickvian**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
close #47181

### 💡 Background and solution
Provided code example in the 
https://ant.design/components/breadcrumb#use-with-browserhistory
doesn't make sense, upon inspection there are few error:

- path for home and second doesn't have /
- in itemRender function variable `item` is being used without being declared or passed from anywhere, the logic to check isLast simply wrong.
- upon testing myself, need to add "/" ahead of paths.join("/") else it will treated as relative path that can cause wrong URL access
- 

### 📝 Changelog

fix non-working code breadcrumb - usage with browserhistory documentation

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix non-working example code in breadcrumb |
| 🇨🇳 Chinese | 修复在面包屑中示例代码无法正常运行的问题 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/breadcrumb/index.zh-CN.md [hunk 1]
```diff
@@ -118,13 +118,19 @@ const items = [
...
-  const last = items.indexOf(item) === items.length - 1;
-  return last ? <span>{item.title}</span> : <Link to={paths.join('/')}>{item.title}</Link>;
+
+function itemRender(currentRoute, params, items, paths) {
+  const isLast = currentRoute?.path === items[items.length - 1]?.path;
+
+  return isLast ? (
+    <span>{currentRoute.title}</span>
+  ) : (
+    <Link to={"/" + paths.join("/")}>{currentRoute.title}</Link>
```
Comment (kiner-tang) [inline components/breadcrumb/index.zh-CN.md:132]:
```suggestion
    <Link to={`/${paths.join("/")}`}>{currentRoute.title}</Link>
```

**Comment by rickvian** [comment]
> ## 👁 Visual Regression Report for PR #47581 Failed ❌
> > **🎯 Target branch:** master ([2c029ea](https://github.com/ant-design/ant-design/commit/2c029eacada8446da060ca3953400a2f2c02ab07))
> > 📖 [View Full Report ↗︎](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-47581/visualRegressionReport/report.html)
> 
> Image name	Expected	Actual	Diff
> `qr-code-Popover.dark.css-var.png`	master: 2c029eacada8446da060ca3953400a2f2c02ab07	current: pr-47581	diff
> Check [Full Report](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-47581/visualRegressionReport/report.html) for details

uh? is the test flaky? i believe it doesn't change any component visual

**Comment by yoyo837** [comment]
> uh? is the test flaky? i believe it doesn't change any component visual

Try rebase upstream.

**Review by kiner-tang — APPROVED** [review]

**Comment by kiner-tang** [comment]
The CI Broken was not caused by this PR.

**Comment by rickvian** [comment]
Thank you!

**Pull request merged**